packages/ra-ui-materialui/src/input/AutocompleteInput.stories.tsx

```diff
@@ -504,6 +504,36 @@ export const CreateItemLabel = () => (
     </Wrapper>
 );
 
+const NoCreateItemLabelInput = () => {
+    const [choices, setChoices] = useState(choicesForCreationSupport);
+    return (
+        <AutocompleteInput
+            source="author"
+            choices={choices}
+            onCreate={async filter => {
+                if (!filter) return;
+
+                const newOption = {
+                    id: choices.length + 1,
+                    name: filter,
+                };
+                setChoices(options => [...options, newOption]);
+                // Wait until next tick to give some time for React to update the state
+                await new Promise(resolve => setTimeout(resolve));
+                return newOption;
+            }}
+            createItemLabel="" // TODO - FIX THIS
+            createLabel="Click to create a new item"
+        />
+    );
+};
```

Comment:
I believe this use case would be nice for people still wanting to start the creation workflow with the `createLabel` and not the `createItemLabel`. All they would need to do would be to pass `createItemLabel=""`.
So I'll see if I can make it work (as it currently doesn't).